templates/zerver/emails/missed_message.html

```diff
@@ -33,6 +33,8 @@
     {% trans %}You are receiving this because everyone was mentioned in #{{ stream_name }}.{% endtrans %}<br />
     {% elif stream_email_notify %}
     {% trans %}You are receiving this because you have email notifications enabled for #{{ stream_name }}.{% endtrans %}<br />
+    {% elif followed_topic_email_notify %}
+    {% trans %}You are receiving this because you have email notifications enabled for the followed topic #{{ stream_name }} > {{ topic_name }}.{% endtrans %}<br />
```

Comment:
@alya thoughts on this string? I think it could be just "... because you are following #{{ stream_name }} > {{topic_name}}.", or "because you have email notifications enabled for topics you follow."; I think the latter might be preferable, since it implies pretty clearly that you follow this topic anyway, with the extra length/clutter of spelling out the topic.

Reply:
Yeah, I like the latter option.

Reply:
Updated the string.